formula/src/test/java/com/instacart/formula/subjects/KeyUsingListFormula.kt

```diff
@@ -17,7 +18,7 @@ class KeyUsingListFormula :
 
     data class ItemRenderModel(
         val item: String,
-        val onDeleteSelected: () -> Unit
+        val onDeleteSelected: Listener<Unit>
```

Comment:
Does this change mean we need to type our callbacks as `Listener` in RenderModels, or is this change specific to these formulas?

Reply:
When using `onEvent` you will need to type to `Listener` type. For now, there still exists `callback` and `eventCallback` which support old style.